Creating an AWS OIDC resource with session duration fails

**Describe the bug**
I tried to create a `octopusdeploy_aws_openid_connect_account` resource, but it's failing

**Steps to reproduce**

1. Create a resource in Terraform:

```
resource "octopusdeploy_aws_openid_connect_account" "example" {
  name             = "example"
  role_arn         = aws_iam_role.example.arn
  session_duration = 1500
  space_id         = octopusdeploy_space.example.id
}
```

2. Apply changes
3. See error

**Expected behaviour**
An AWS OIDC account to be created in the specified space

**Logs and other supporting information**
```
Stack trace from the terraform-provider-octopusdeploy_v0.18.1 plugin:

panic: interface conversion: interface {} is int, not string

goroutine 113 [running]:
github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy.expandAmazonWebServicesOpenIDConnectAccount(0xc00080b100)
github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy/schema_amazon_web_services_openid_connect_account.go:57 +0x4c5
github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy.resourceAmazonWebServicesOpenIDConnectAccountCreate({0x1044e58, 0xc0004a4bd0}, 0xc00080b100, {0xdef540?, 0xc000182840})
github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy/resource_aws_openid_connect_account.go:27 +0x4c
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc0005789a0, {0x1044db0, 0xc000689440}, 0xd?, {0xdef540, 0xc000182840})
github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/resource.go:707 +0x11b
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc0005789a0, {0x1044db0, 0xc000689440}, 0xc0007c8750, 0xc00080af80, {0xdef540, 0xc000182840})
github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/resource.go:837 +0xa69
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc0002f4c48, {0x1044db0?, 0xc000689350?}, 0xc0007cc870)
github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/grpc_provider.go:1021 +0xdbc
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc00011e3c0, {0x1044db0?, 0xc000688240?}, 0xc0004a4230)
github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/tf5server/server.go:818 +0x56b
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xe8fde0?, 0xc00011e3c0}, {0x1044db0, 0xc000688240}, 0xc0004a41c0, 0x0)
github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x169
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00043c000, {0x104b808, 0xc000581040}, 0xc000960000, 0xc000459d70, 0x16f3520, 0x0)
google.golang.org/grpc@v1.58.3/server.go:1374 +0xde7
google.golang.org/grpc.(*Server).handleStream(0xc00043c000, {0x104b808, 0xc000581040}, 0xc000960000, 0x0)
google.golang.org/grpc@v1.58.3/server.go:1751 +0x9e7
google.golang.org/grpc.(*Server).serveStreams.func1.1()
google.golang.org/grpc@v1.58.3/server.go:986 +0xbb
created by google.golang.org/grpc.(*Server).serveStreams.func1 in goroutine 59
google.golang.org/grpc@v1.58.3/server.go:997 +0x145

Error: The terraform-provider-octopusdeploy_v0.18.1 plugin crashed!
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment and versions:**
- OS: Terraform Cloud (Linux?)
- Octopus Server Version: 2024.2 (Build 6366)
- Terraform Version: 1.7.5
- Octopus Terraform Provider Version: 0.18.1

**Additional context**
Add any other context about the problem here.

--- a/octopusdeploy/schema_amazon_web_services_openid_connect_account.go
+++ b/octopusdeploy/schema_amazon_web_services_openid_connect_account.go
@@ -7,6 +7,7 @@ import (
 	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/core"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"strconv"
 )
 
 func expandAmazonWebServicesOpenIDConnectAccount(d *schema.ResourceData) *accounts.AwsOIDCAccount {
@@ -53,7 +54,7 @@ func expandAmazonWebServicesOpenIDConnectAccount(d *schema.ResourceData) *accoun
 	}
 
 	if v, ok := d.GetOk("session_duration"); ok {
-		account.SessionDuration = v.(string)
+		account.SessionDuration = strconv.Itoa(v.(int))
 	}
 
 	return account

